test/test_sendfile.rs

```diff
@@ -1,15 +1,23 @@
 use std::io::prelude::*;
 use std::os::unix::prelude::*;
 
-use tempfile::tempfile;
-
 use libc::off_t;
+use nix::sys::sendfile::*;
+use tempfile::tempfile;
 
-use nix::unistd::{close, pipe, read};
-use nix::sys::sendfile::sendfile;
+cfg_if! {
+    if #[cfg(any(target_os = "linux", target_os = "android"))] {
+        use nix::unistd::{close, pipe, read};
+    }
+    else if #[cfg(any(target_os = "freebsd", target_os = "ios", target_os = "macos"))] {
```

Comment:
Please put `else` expressions on the same line as the closing brace as suggested by [the style guidelines](https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/expressions.md#control-flow-expressions).